releasenotes/notes/otlp-v0.59.0-be24aaaa593938c7.yaml

```diff
@@ -8,4 +8,4 @@
 ---
 enhancements:
   - |
-    The OTLP ingest endpoint now supports the same settings and protocol as the OpenTelemetry Collector OTLP receiver v0.59.0. In particular, this drops support for consuming OTLP/JSON v0.15.0 or below payloads.
+    The OTLP ingest endpoint now supports the same settings and protocol as the OpenTelemetry Collector OTLP receiver v0.60.0. In particular, this drops support for consuming OTLP/JSON v0.15.0 or below payloads.
```

Comment:
I think this was meant to be a new file, not overwrite the v0.59.0 file?

Reply:
No, this was intentional, I should have explained somewhere, sorry! The PR that introduced that note (#13395) is also for 7.40. If we have two notes the changelog will say something like:

> - The OTLP ingest endpoint now supports the same settings and protocol as the OpenTelemetry Collector OTLP receiver v0.59.0
> - The OTLP ingest endpoint now supports the same settings and protocol as the OpenTelemetry Collector OTLP receiver v0.60.0

which would be confusing, so I decided the best was to edit the existing release note.

Reply:
I need to merge this before the freeze. Please have a look at the explanation, if something is wrong we can fix it in a follow up PR, thanks!